src/interfaces/chain.cpp

```diff
@@ -25,6 +26,8 @@
 #include <memory>
 #include <utility>
 
+class CReserveScript;
```

Comment:
unnecessary given this is declared in the header

Reply:
Aside: Perhaps IWYU can be included in `doc/developer-notes.md`

Reply:
Agree, and it should come with steps to install for Bitcoin Core.

Reply:
> Agree, and it should come with steps to install for Bitcoin Core.

It's not very convenient, but here are steps I've followed previously: https://github.com/bitcoin/bitcoin/pull/11878#issuecomment-351610027